src/components/ImageDetails/PhotoDetails.vue

```diff
@@ -236,7 +236,7 @@ export default {
         : ''
     },
     ccLicenseURL() {
```

Comment:
```suggestion
    openverseLicenseURL() {
```

Reply:
Could you also please change the prop name from `ccLicenseURL` to `openverseURL`? On lines 130, 145, and inside the corresponding components.

Reply:
yes sure openverseURL or openverseLicenseURL? @obulat